kuksa-client/kuksa_client/cli_backend/grpc.py

```diff
@@ -127,28 +138,33 @@ def setValues(self, updates: Dict[str, Any], attribute="value", timeout=5):
         if attribute in self.AttrDict:
             field, _ = self.AttrDict[attribute]
             entry_updates = []
+            v1 = True
             for path, value in updates.items():
 
                 if field is kuksa_client.grpc.Field.VALUE:
                     entry = kuksa_client.grpc.DataEntry(
-                        path=path, value=kuksa_client.grpc.Datapoint(value=value))
+                        path=path,
+                        value=kuksa_client.grpc.Datapoint(value=value),
+                    )
+                    v1 = False
```

Comment:
Maybe we need a comment or a better variable name here, like `try_v1` (or maybe reverse it, i.e. `try_v2 = True`). As I understand it the meaning of `v1` here is to indicate whether we will try v2 or not.